app/views/admin/users/_user_and_products.html.erb

```diff
@@ -13,11 +13,11 @@
   <input type="radio" name="user-tabs" id="products-tab-radio" <%= "checked" if show_products_tab %> style="display: none;">
 
   <!-- Tab buttons -->
-  <div role="tablist" class="tab-buttons" aria-label="Profile Tabs">
-    <label for="profile-tab-radio" role="tab" style="cursor: pointer;">
+  <div role="tablist" class="tab-container grid grid-auto-cols-fr gap-3 sm:grid-flow-col" aria-label="Profile Tabs">
```

Comment:
added a `tab-container` class just to reference in the selectors used in this file